packages/paste-website/src/pages/components/display-pill-group/index.mdx

```diff
@@ -0,0 +1,232 @@
+---
+title: Display Pill Group
+package: '@twilio-paste/display-pill-group'
+description: A Display Pill Group is a set of Pills used to visually represent a collection of entities outside of form-based UIs.
+slug: /components/display-pill-group/
+---
+
+import {graphql} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {DisplayPill, DisplayPillGroup} from '@twilio-paste/display-pill-group';
+import Changelog from '@twilio-paste/display-pill-group/CHANGELOG.md';
+import {CalendarIcon} from '@twilio-paste/icons/esm/CalendarIcon';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {FormPillVsDisplayPillTable} from '../../../components/FormPillVsDisplayPillTable';
+import {SidebarCategoryRoutes} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/display-pill-group"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/components/display-pill-group/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Display Pill Group"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/display-pill-group"
+  storybookUrl="/?path=/story/components-display-pill-group--basic"
+  data={props.data.allPasteComponent.edges}
+  packageStatus={props.data.allAirtable.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview scope={{DisplayPill, DisplayPillGroup, CalendarIcon}}>
+  {`<DisplayPillGroup aria-label="Your favorite sports:">
+  <DisplayPill href="https://google.com">
+    <CalendarIcon decorative size="sizeIcon10" />
+    Tennis
+  </DisplayPill>
+  <DisplayPill>Football</DisplayPill>
+  <DisplayPill href="/">Baseball</DisplayPill>
+  <DisplayPill>Basketball</DisplayPill>
+  <DisplayPill>Soccer</DisplayPill>
+</DisplayPillGroup>`}
+</LivePreview>
+
+## Guidelines
+
+### About Display Pill Group
+
+<Paragraph>
+  {props.pageContext.frontmatter.description} Display Pills are static and should be used in situations where data is
+  not being modified.
+</Paragraph>
+
+Display Pills can link to other pages. This can useful when the entity the pill represents has it's own detail page within the application.
```

Comment:
```suggestion
Display Pills can link to other pages. This can useful when the entity the pill represents has its own detail page within the application.
```